docs/sources/tempo/operations/dedicated_columns.md

```diff
@@ -2,8 +2,13 @@
 title: Dedicated columns
 description: Learn how to use dedicated columns to improve query performance.
 weight: 42
+_build:
```

Comment:
i don't see this page in 2.2. I think it's only in tip of main so this directive is not necessary. Screenshot of the 2.2 "Manage" menu:

![image](https://github.com/grafana/tempo/assets/2272392/2da29aec-704f-43eb-aa31-ebc875e70b55)